projects/src/transmitter.cc

```diff
@@ -0,0 +1,157 @@
+#include "transmitter.h"
+#include "hal.h"
+#include "packet/serialize.h"
+#include "usbconfig.h"
+
+// This define can be useful when sizing message mailbox and memory pools
+#define ASSERT_MESSAGE_MEMORY_LIMIT FALSE
```

Comment:
If that is the purpose of the flag, I suggest renaming it to something like `OVERWRITE_OLD_PACKETS_TO_PREVENT_OVERFLOW` or something.